bin/lesson_check.py

```diff
@@ -558,7 +570,8 @@ def __init__(self, args, filename, metadata, metadata_len, text, lines, doc):
     (re.compile(r'index\.md'), CheckIndex),
     (re.compile(r'reference\.md'), CheckReference),
     (re.compile(r'_episodes/.*\.md'), CheckEpisode),
-    (re.compile(r'.*\.md'), CheckGeneric)
+    (re.compile(r'.*\.md'), CheckGeneric),
+    (re.compile(r'.*\.snip'), CheckNonJekyll)
```

Comment:
I deliberately chose `CheckNonJekyll` otherwise we would have to add yaml metadata to all the snippets (which make no sense)